app/models/project_mediaflux.rb

```diff
@@ -77,6 +78,17 @@ def self.create_accumulators(mediaflux_project_id:, session_id:)
     accum_size.resolve
   end
 
+  def self.create_quota(project:, mediaflux_project_id:, session_id:)
+    #TODO: SHOULD WE CREATE A PROJECT USING REQUESTED VALUES OR APPROVED VALUES?
```

Comment:
Good question.  Should we bring this up to the team?  No approval step happens, so I would assume the requested values like you did here.

Reply:
Yea I think it's a good idea to bring this up during the next stand-up